lib/src/base-styles/_nx-page-header.scss

```diff
@@ -63,7 +64,7 @@
   }
 
   &:active, &.nx-page-header__link--current {
-    border-bottom-color: var(--nx-color-link);
+    border-bottom-color: var(--nx-color-page-header-active-link-border-bottom);
```

Comment:
This one is just a small nitpick, after renaming that variable we can use `border-color` here.

Reply:
> This one is just a small nitpick, after renaming that variable we can use `border-color` here.

Changed here: [2f60ae7](https://github.com/sonatype/sonatype-react-shared-components/pull/979/commits/2f60ae73414a86d602d6aabcbc64187faef4e5b2)